sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsView.java

```diff
@@ -225,7 +226,7 @@ public EventsView() {
 			createButton = ButtonHelper.createIconButton(
 				Captions.eventNewEvent,
 				VaadinIcons.PLUS_CIRCLE,
-				e -> ControllerProvider.getEventController().create(null),
+				e -> ControllerProvider.getEventController().create((CaseReferenceDto) null),
```

Comment:
What's the purpose of adding this cast?

Reply:
I overloaded the create method, and therefore he can't choose between the two if the argument isn't typed.